guides/common/modules/proc_configuring-for-uefi-http-boot-ipv6-provisioning.adoc

```diff
@@ -0,0 +1,39 @@
+[id="configuring-for-uefi-http-boot-ipv6-provisioning_{context}"]
+= Configuring {Project} for UEFI HTTP Boot IPv6 Provisioning
+
+Use this procedure to configure {Project} to provision hosts in an IPv6 network with UEFI HTTP Boot provisioning.
+
+.Prerequisites
+
+* Ensure that your clients can access DHCP and TFTP servers.
+
+* Ensure that the UDP ports 67 and 68 are accessible by the client so that the client can send and receive a DHCP request and offer.
+
+* Ensure that the TCP port 8000 is open for the client to download files and Kickstart templates from {SmartProxy}.
+
+* Ensure that the host provisioning interface subnet has a TFTP {SmartProxy}, and Templates {SmartProxy} set. For more information, see https://access.redhat.com/documentation/en-us/red_hat_satellite/6.7/html/provisioning_guide/configuring_networking#Configuring_Networking-Adding_a_Subnet_to_the_Satellite_Server[Adding a Subnet to {ProjectServer}] in the _Provisioning Guide_.
+
+.Procedure
+
+. Since clients that are booting from network cannot be identified by remote IPv6 address due to unmanaged DHCPv6 service, you must enable provisioning tokens. To enable provisioning tokens, navigate to *Administer* > *Settings* > *Provisioning* and set the *Token duration* setting to 0.
+
+. You must disable DHCP management in the installer or not use it.
+
+. For all IPv6 subnets created in {Project}, set the *DHCP {SmartProxy}* to blank.
+
+. Optional: If the host and the DHCP server are separated by a router, configure the DHCP relay agent and point to the DHCP server.
+
+. Update the `grub2-efi` package to the latest version:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {package-install-project} grub2-efi
+----
+
+. Optional: If native IPv6 connectivity is not available, configure an HTTP proxy in {Project} and set it as the default proxy in settings. For more information, see https://access.redhat.com/documentation/en-us/red_hat_satellite/6.8-beta/html/installing_satellite_server_from_a_connected_network/performing-additional-configuration#adding-a-default-http-proxy_satellite[Adding a Default HTTP Proxy to Satellite].
```

Comment:
Is there a way to link to this content internally, without linking to the RH docs?

Reply:
This is being discussed in the issue #192. In this case, this module is likely to be reused, and if it is reused in another guide, xref won't work. Hence the link.

Reply:
Let's continue the discussion there, we have many of these and this needs to be solved somehow.